gocourse05/go.mod

```
@@ -0,0 +1,3 @@
+module gocourse05
+
+go 1.22.6
```

Варто використовувати найновішу версію go 1.23.0
```suggestion
go 1.23.0
```

---

https://go.dev/dl/